deeplake/core/linked_chunk_engine.py

```diff
@@ -211,7 +211,11 @@ def get_basic_sample(self, global_sample_index, index, fetch_chunks=False):
         sample = self.get_deeplake_read_sample(global_sample_index, fetch_chunks)
         if sample is None:
             return np.ones((0,))
-        return sample.array[tuple(entry.value for entry in index.values[1:])]
+        arr = sample.array
+        max_shape = self.tensor_meta.max_shape
+        if len(arr.shape) == 2 and max_shape and len(max_shape) == 3:
```

Comment:
why do we check and max_shape, can it be empty? if yes `len(max_shape)` will fail

Reply:
if it is None, it will short circuit at max_shape check and then len(max_shape) won't be called